frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -121,6 +121,11 @@ history-history-no-translations = No translations available.
 history-translation-copy =
     .title = Copy Into Translation (Tab)
 
+history-translation-hide-diff = Hide diff
+    .title = Hide diff against the currently active translation
+history-translation-show-diff = Show diff
+    .title = Show diff against the currently active translation
+
```

Comment:
@adngdb Could you please add fr (and es?) translations? :)

Reply:
Yo no hablo español. :cry:

Reply:
French translation: https://github.com/mathjazz/pontoon/pull/8